test/state/state.cpp

```diff
@@ -123,6 +123,9 @@ void finalize(State& state, evmc_revision rev, const address& coinbase,
     if (block_reward.has_value())
         state.touch(coinbase).balance += *block_reward;
 
+    for (const auto& withdrawal : withdrawals)
+        state.touch(withdrawal.recipient).balance += withdrawal.get_amount();
```

Comment:
I don't remember why we implemented this in different order originally. The use of `.touch()` indicates we want the account to be cleared.